docs/sql-ref-syntax-qry-select-hints.md

```diff
@@ -51,6 +51,10 @@ specified, multiple nodes are inserted into the logical plan, but the leftmost h
 
   The `REPARTITION_BY_RANGE` hint can be used to repartition to the specified number of partitions using the specified partitioning expressions. It takes column names and an optional partition number as parameters.
 
+* **REPARTITION_BY_AQE**
```

Comment:
Can we describe it in more details? I don't think any of users is able to use this hint after reading this docs.

Reply:
Also what happens if AQE is disabled?

Reply:
The doc here is pretty bad...

```
**REBALANCE_PARTITIONS** 

The `REBALANCE_PARTITIONS` hint can be used to rebalance the query result output partitions, so that
every partition is of a reasonable size (not too small and not too big). It can take column names as
parameters, and try its best to partition the query result by these columns. This is a best-effort: if
there are skews, Spark will split the skewed partitions, to make these partitions not too big.
This hint is ignored if AQE is not enabled.
```

Reply:
so sorry for this, I prepare to write more docs after PR [#32883](https://github.com/apache/spark/pull/32883) ..

Reply:
> This hint is ignored if AQE is not enabled.

@cloud-fan do you mean we should remove this operator if AQE is disabled ?

Reply:
Or fallback to `repartition(col)`? I don't have a strong preference

Reply:
`REBALANCE_PARTITIONS` has no meaning without AQE, +1 for ignore.